### PR TITLE
New material for multiprocessing using the native multiprocessing package

### PR DESCRIPTION
**Pull request by pletzer**

### CODE REVIEW

File: _lessons/python-scatter/04a-multiprocessing.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+Multiprocessing is suitable when you have a large number of tasks to be executed in any order.
+
+### Pros
+
+ * A way to leverage multiple CPU cores for increased performance
+ * Can handle different work loads
+
+### Cons
+
+ * All processes must share memory - not suitable on different cluster nodes
```
Comment (chrisdjscott) [inline _lessons/python-scatter/04a-multiprocessing.md]:
```suggestion
 * Typically, all processes must run within the same node
```

Comment (chrisdjscott) [reply _lessons/python-scatter/04a-multiprocessing.md]: @pletzer see above for suggested change

**Comment by pletzer** [comment]
Hi @chrisdjscott 

I changed the text to 
"""
### Cons
  * Processes must typically run on the same node
"""
How does that sound?

**Comment by chrisdjscott** [comment]
Looks good

**Pull request merged**